### PR TITLE
fix: improve  `avatar` section of README.md

### PR DESCRIPTION
**Pull request by Rahulbaran**
Mentioned it clearly that someone just have to replace its `<yourusername>`  with its **gh username** in the URL to include `avatar`.

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/930"></a>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,7 +98,9 @@ Your URL will be `http://linkfree.eddiehub.org/<yourusername>`
 
 For example: <http://linkfree.eddiehub.org/eddiejaoude>
 
-Your `avatar` URL should take the format of `https://github.com/<yourusername>.png`
+To include your `avatar`, You just have to replace `<yourusername>` with your **github username** in the format of `https://github.com/<yourusername>.png` and it will automatically be fetched from github account. 
+
+For example: <http://linkfree.eddiehub.org/eddiejaoude.png>
```
Comment (krishguptadev) [inline README.md:103]:
```suggestion
For example: <http://github.com/eddiejaoude.png>
```

**Review by krishguptadev — CHANGES_REQUESTED** [review]

**Comment by Rahulbaran** [comment]
One check is pending, can anyone guide me what's the reason??

**Review by krishguptadev — APPROVED** [review]

**Review by emmalearnscode — APPROVED** [review]

**Comment by emmalearnscode** [comment]
> One check is pending, can anyone tell me what's the reason??

We’re not sure right now, but don’t worry about it as it doesn’t directly affect your code :)

**Review by Cahllagerfeld — APPROVED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
LGTM :) Thanks for the contribution

**Comment by Rahulbaran** [comment]
Thank you all for merging my PR🤗

**Pull request merged**